spec/shadow/index.html

```diff
@@ -1137,6 +1137,20 @@
       </section>
 
       <section>
+        <h3>Focus</h3>
+      <p>A <a>shadow host</a> can delegate focus to its <a>shadow root</a> by assigning a boolean <a href="#widl-ShadowRootInit-delegatesFocus">delegatesFocus</a> flag to be true in <a href="#idl-def-ShadowRootInit">ShadowRootInit</a> dictionary. If omitted, a <a>shadow host</a> does not delegate focus to its shadow root, and the shadow host itself can be focusable.</p>
```

Comment:
Could you surround "delegate focus" by `<dfn></dfn>`? And reference it in other places.